package.json

```diff
@@ -6,7 +6,7 @@
   "main": "./index.js",
   "repository": {
     "type": "git",
-    "url": "https://github.com/mapd/mapd-crossfilter.git"
+    "url": "https://github.com/sachinkaushikHERE/mapd-crossfilter.git"
```

Comment:
let's make sure not to land this with this changed

Reply:
Done